jenkins/docker/ubuntu/javascript.Dockerfile

```diff
@@ -17,11 +17,11 @@ RUN apt-get install -y wget gnupg \
 
 # nodejs
 ENV NODE_OPTIONS="--dns-result-order=ipv4first"
-ARG NODE_MAJOR=18
+ARG NODEJS_VERSION=18
```

Comment:
fyi - 18 is in maintenance, maybe we should look into upgrading to active (20) in separate PR

Reply:
Yeah, I will also update the default value.  This will be overwritten by values in the ``baseImages.yaml``